spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/orm/jpa/HibernateProperties.java

```diff
@@ -35,6 +35,7 @@
  * Configuration properties for Hibernate.
  *
  * @author Stephane Nicoll
+ * @author Chris Bono
```

Comment:
Do I need to add myself as author here when its 2 lines of code? I mean, it does add change in behavior so I can see either way. I can't remember the threshold when to add to author tag.